src/ui/actions/timeline.ts

```diff
@@ -191,6 +192,20 @@ export function setTimelineToTime(time: number | null, updateGraphics = true): U
   };
 }
 
+export function setTimelineToPauseTime(
+  pauseId: string,
+  time: number | null,
+  updateGraphics = true
+): UIThunkAction {
+  return async dispatch => {
+    dispatch(setTimelineToTime(time, updateGraphics));
+
+    if (time) {
+      dispatch(() => repaintAtPause(time, pauseId));
```

Comment:
@markerikson - this might be a dumb way to add a one-off dispatch. I'll claim ignorance because I haven't used redux. I _think_ I want it to be scheduled by the dispatcher so I know it's called after the `setTimelineToTime` but maybe this doesn't guarantee that. Advice welcome!

Reply:
I think this was bad but needed to take a different approach anyway.